index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang= "en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portofolio</title>
+    <link rel="stylesheet" href="style.css">
+    </head>
+<body>
+<header class="Header">
+     <a class="mylogo">MY logo</a>
+             <nav>
+            <a href="#"> </a>
+            <img href src="./image/logobuger.JPG">
+     </nav>
+</header>
+<main class="Workssection1">
+       
+        <div class="mainiam">
+            <h1 class="introdaction"> I'm Mihertab Glad to see you!</h1>
+        </div>
+        <div class="section_01">
+            <p class="about"> I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! If you like what 
+                you see and have a project you need coded, don’t hestiate to contact me. </p>
+        </div>
+        <div class="letconnect">
+            <h4 class="connectmywork"> LET’S CONNECT</h4>
+        </div>
+        <div class = "social-section">
+                <ul class="social-icons-mywork">
+                    <li><a href="#"><img src="./image/Vectortweet.png" alt=""></a></li>
+                    <li><a href="#"><img src="./image/Linkedin.svg" alt=""></a></li>
+                    <li><a href="#"><img src="./image/VectorM.jpg" alt=""></a></li>
+                    <li><a href="#"><img src="./image/VectorGit.png" alt=""></a></li>
+                    <li><a href="#"><img src="./image/Vector5.png" alt=""></a></li>
+                    
+                </ul>
+        </div>
+        
+      
+</main>
```

Comment:
- [ ] As requested by the previous reviewer please replace the `main` tag with the `section` tag because It represents a generic standalone section of the document.